src/main/webapp/navigation-bar.jsp

```diff
@@ -1,7 +1,7 @@
 <div id="navigation">
   <ul>
     <li><a href="/home">Home</a></li>
-    <li><a href="/task?taskID=1">Task Page</a></li>
+    <li><a href="/task?taskID=0">Task Page</a></li>
```

Comment:
What's the significance of taskID=0?  If this is just for preventing lookups if a valid taskID was not provided, the servlet should be the one checking for the validity of the taskID and skip the relevant steps if the taskID is not provided